administracion/views.py

```diff
@@ -1,40 +1,69 @@
 from django.contrib.auth.models import User
 from django.shortcuts import render, redirect
 from django.contrib.auth.decorators import user_passes_test, login_required
-from .forms import FormaCreacionUsuario
+
+from .forms import UserForm
 from perfiles_usuario.utils import is_administrador
 
 
-@login_required
-@user_passes_test(is_administrador)
+@login_required(login_url='tosp_auth:login')
```

Comment:
None of these login_url are required, since the default login_url was inserted in settings with PR #38